.github/github-functions.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+function get_infection_pr_flags() {
+    local flags="";
+    local changed_files;
+
+    PR_NUMBER=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')
+
+    if ! [[ "${PR_NUMBER}" == "" ]]; then
+        git remote set-branches --add origin "$GITHUB_HEAD_REF";
+        git fetch;
+
+        changed_files=$(git diff master --diff-filter=A --name-only | grep src/ | paste -sd "," -);
+
+        if [ -n "$changed_files" ]; then
+            # Set those flags only if there is any changed files detected
+            flags="--filter=${changed_files} --ignore-msi-with-no-mutations --only-covered ${flags}";
```

Comment:
```suggestion
            flags="--filter=${changed_files} --ignore-msi-with-no-mutations --min-msi=100.0 --min-covered-msi=100.0 --only-covered ${flags}";
```

This report is primarily important in case of CI failure. But if we make Infection fail the CI, guess what, the next command with `cs2pr` won't run!

Reply:
no worries, I like your idea ;) it makes sense

but I'm not sure about `--min-msi=100.0 --min-covered-msi=100.0` in the comment above. Do we really want to fail the build even with 1 escaped mutant?

well, we can try...

Reply:
Surely not, but for the sake of an example! The whole schema should work even if Infection fails.